compose/material3/material3/build.gradle

```diff
@@ -104,7 +104,8 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
                 api("org.jetbrains.compose.runtime:runtime:1.8.1")
                 api("org.jetbrains.compose.ui:ui-graphics:1.8.1")
                 api("org.jetbrains.compose.ui:ui-text:1.8.1")
-                api(project(":graphics:graphics-shapes"))
+                api("org.jetbrains.compose.ui:ui-text:1.8.1")
+                api("androidx.graphics:graphics-shapes:1.0.1")
```

Comment:
We should still use binary from the fork. Pin to latest released alpha.

Currently CI should fail because it doesn't support web targets